app/input_objects/pages/long_lists_selection/bulk_options_input.rb

```diff
@@ -0,0 +1,49 @@
+class Pages::LongListsSelection::BulkOptionsInput < BaseInput
+  DEFAULT_OPTIONS = { selection_options: [],
```

Comment:
I don't think this is used anywhere, we could get rid of it.

Reply:
Done!